sdg/stats.py

```diff
@@ -43,8 +43,10 @@ def reporting_status(schema, all_meta, extra_fields=None):
     meta_df = pd.DataFrame(rows, index=all_meta.keys())
 
     # Make sure that numeric columns are numeric.
+    def x_is_numeric(x):
```

Comment:
I would suggest not naming this function `x_is_numeric`.
```suggestion
    def value_is_numeric(value):
```

Also, do we need to keep decimal values into account? If so , Python's `isnumeric()` returns `False` when the value is decimal (for example `55.1`)

Reply:
@SavvasStephanides Thanks, I changed that function name/parameter.

In this case I don't think we need to worry about floats. This bit of code would be used to group indicators into different categories, according to the values of metadata fields. That integer stuff was to support countries that might be using integers to categorize indicators. (Eg, 1 = national, 2 = global, etc.) In this case, we just wanted to make sure that the data type was consistent, so that all the 1's would be grouped together (for example) in case somehow Python was reading some 1's as strings and other 1's as integers. I guess that could happen if, in the YAML, there was inconsistency between `1` and `'1'`. I don't think that anyone would be using floats to categorize indicators. (To be honest, I can't remember whether that bit of code was a response to an actual problem, or just a "maybe this will be a problem" kind of thing.)

Reply:
OK. Thanks for clarifying!